logging.go

```diff
@@ -0,0 +1,101 @@
+package dht
+
+import (
+	"fmt"
+	"github.com/multiformats/go-multibase"
+	"github.com/multiformats/go-multihash"
+	"strings"
+
+	"github.com/ipfs/go-cid"
+	"github.com/multiformats/go-base32"
+)
+
+func tryFormatLoggableRecordKey(k string) (string, error) {
+	if len(k) == 0 {
+		return "", fmt.Errorf("loggableRecordKey is empty")
+	}
+	var proto, cstr string
+	if k[0] == '/' {
+		// it's a path (probably)
+		protoEnd := strings.IndexByte(k[1:], '/')
+		if protoEnd < 0 {
+			return "", fmt.Errorf("loggableRecordKey starts with '/' but is not a path: %s", base32.RawStdEncoding.EncodeToString([]byte(k)))
+		}
+		proto = k[1 : protoEnd+1]
+		cstr = k[protoEnd+2:]
+
+		var encStr string
+		c, err := cid.Cast([]byte(cstr))
+		if err == nil {
+			encStr = c.String()
+		} else {
+			encStr = base32.RawStdEncoding.EncodeToString([]byte(cstr))
+		}
```

Comment:
@petar  and I talked about this and it's probably more sensible to just output `/pathprefix/base32encodedData` without any processing of whether it's a CID or not.